.vscode/settings.json

```diff
@@ -8,6 +8,5 @@
 	"editor.codeActionsOnSave": {
 		"source.fixAll.eslint": true
 	},
-	"editor.formatOnSave": true,
```

Comment:
I find the formatting on save to be pretty helpful. I'm curious why you removed it?

Reply:
I feel like all settings should be in .gitignore. For me, it was causing problems with the linter. Other people's setups might be different, no?

Reply:
Good points, this should be solved by #12

Reply:
When you get a chance, could you sync with the main branch and remove the VS Code settings related changes from this PR?